package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "googlemaps",
+  "version": "0.1.0",
+  "description": "googlemaps",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "pretest": "tsc",
+    "test": "karma start --single-run && node tests/remap.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendixlabs/google-maps"
+  },
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/mendixlabs/google-maps/issues"
+  },
+  "devDependencies": {
+    "@types/big.js": "0.0.31",
```

Comment:
Discussed this issue with Johan Flint; as the types are not used in the runtime they should not be part of the dependencies, as they are only used during development.